### PR TITLE
CORE-19290 - Replaced the UTCClock by AutoTickTestClock in the tests and removed the sleep calls

### PR DESCRIPTION
**Pull request by filipesoliveira**

### CODE REVIEW

**Review by vkolomeyko — APPROVED** [review]
LGTM

File: components/ledger/ledger-utxo-token-cache/src/test/kotlin/net/corda/ledger/utxo/token/cache/impl/entities/TokenCacheImplTest.kt [hunk 1]
```diff
@@ -85,42 +86,32 @@ class TokenCacheImplTest {
...
     fun `ensure the expiry period is not applied if the random strategy is used`() {
-        val expiryPeriodInMillis = 5L
         val strategy = Strategy.RANDOM
 
-        val target = TokenCacheImpl(Duration.ofMillis(expiryPeriodInMillis), UTCClock())
+        val target = TokenCacheImpl(1.seconds, AutoTickTestClock(Instant.EPOCH, 2.seconds))
         target.add(listOf(cachedToken1, cachedToken2, cachedToken3), strategy)
         assertThat(target.get(strategy).toList()).containsOnly(cachedToken1, cachedToken2, cachedToken3)
-        sleep(expiryPeriodInMillis)
-        assertThat(target.get(strategy).toList()).containsOnly(cachedToken1, cachedToken2, cachedToken3)
```
Comment (josephzunigadaly) [inline components/ledger/ledger-utxo-token-cache/src/test/kotlin/net/corda/ledger/utxo/token/cache/impl/entities/TokenCacheImplTest.kt:108]: We want this assert, to check that the expiry is not applied

File: components/ledger/ledger-utxo-token-cache/src/test/kotlin/net/corda/ledger/utxo/token/cache/impl/entities/TokenCacheImplTest.kt [hunk 2]
```diff
@@ -85,42 +86,32 @@ class TokenCacheImplTest {
...
         assertThat(target.get(strategy).toList()).isEmpty()
     }
 
     @Test
     fun `ensure the expiry period is not applied if the random strategy is used`() {
-        val expiryPeriodInMillis = 5L
         val strategy = Strategy.RANDOM
 
-        val target = TokenCacheImpl(Duration.ofMillis(expiryPeriodInMillis), UTCClock())
+        val target = TokenCacheImpl(1.seconds, AutoTickTestClock(Instant.EPOCH, 2.seconds))
```
Comment (josephzunigadaly) [inline components/ledger/ledger-utxo-token-cache/src/test/kotlin/net/corda/ledger/utxo/token/cache/impl/entities/TokenCacheImplTest.kt:101]: Closest match to the original would be `5.millis` for both durations in this line

File: components/ledger/ledger-utxo-token-cache/src/test/kotlin/net/corda/ledger/utxo/token/cache/impl/entities/TokenCacheImplTest.kt (hunk 1 above)
Comment (filipesoliveira) [reply components/ledger/ledger-utxo-token-cache/src/test/kotlin/net/corda/ledger/utxo/token/cache/impl/entities/TokenCacheImplTest.kt:108]: The expiry period was set to 1 second and the auto-increment was set to 2 seconds. One assert is enough to check that the expiry is not applied.

Comment (josephzunigadaly) [reply components/ledger/ledger-utxo-token-cache/src/test/kotlin/net/corda/ledger/utxo/token/cache/impl/entities/TokenCacheImplTest.kt:108]: I see. In that case, I'll resolve these comments. Thank you Filipe.

**Review by josephzunigadaly — APPROVED** [review]

**Pull request merged**